pubspec.yaml

```diff
@@ -1,20 +1,19 @@
 name: sass_builder
-version: 2.2.2-dev.0
-
-homepage: https://github.com/dart-league/sass_builder
+version: 2.3.0
+repository: https://github.com/dart-league/sass_builder
 description: Transpile sass files using the "build" package.
 
 environment:
-  sdk: '>=2.15.0 <4.0.0'
+  sdk: ^3.6.0
 
 dependencies:
-  build: ^3.0.0-0
-  build_config: ^1.0.0
+  build: ^3.0.1
+  build_config: ^1.2.0
   path: ^1.8.0
-  sass: ^1.55.0
+  sass: ^1.74.0
```

Comment:
Any reason we're depending on this as the minimum version now? It looks like we're not using any new APIs, or is this the lowest version that resolves against the other dependencies?